scripts/storage-facts.py

```diff
@@ -164,7 +164,7 @@ def generateFacts(original_facts, storage_host):
 
       # Create iSCSI target config, if it doesn't already exist
       if not any(d['name'] == fs_prefix + org + '-' + host for d in facts['iscsi_targets']):
-        target = generateIscsiTarget(host, fs_prefix + org + '/' + host, portals)
+        target = generateIscsiTarget(host, org, fs_prefix + org + '/' + host, portals)
```

Comment:
Maybe it's nicer to leave the generateIscsiTarget function as it is and call instead generateIscsiTarget(org+host, fs_prefix + org + '/' + host, portals)

Reply:
You're right. I added it as parameter because originally, I wanted to leave the backstore name as is.